src/overlays/actors/ovl_Fishing/z_fishing.h

```diff
@@ -9,28 +9,26 @@ struct Fishing;
 typedef struct Fishing {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ char unk_14C[0x004];
-    /* 0x0150 */ u8 unk_150;
-    /* 0x0151 */ u8 unk_151;
+    /* 0x0150 */ u8 isLoach; //rename to fish type?
+    /* 0x0151 */ u8 unk_151; //lily byte
     /* 0x0152 */ u8 unk_152;
     /* 0x0154 */ s16 unk_154;
     /* 0x0156 */ u8 unk_156;
     /* 0x0157 */ u8 unk_157;
     /* 0x0158 */ s16 unk_158;
     /* 0x015A */ s16 unk_15A;
-    /* 0x015C */ s16 unk_15C;
+    /* 0x015C */ s16 state; //fish use as bitfeild, owner as talking state
```

Comment:
I'm not sure this really is a bitfield for fishs

It's ++'ed in UpdateFish and used as-is in calculations